x/msgfees/types/msgs.go

```diff
@@ -50,9 +55,29 @@ func (msg MsgAssessCustomMsgFeeRequest) ValidateBasic() error {
 	if !msg.Amount.IsPositive() {
 		return errors.New("amount must be greater than zero")
 	}
+	_, err = msg.GetBips()
+	if err != nil {
+		return err
+	}
 	return nil
 }
 
+// GetBips converts the msg RecipientBasisPoints to a uint32 basis point value 0 - 10,000
+func (msg MsgAssessCustomMsgFeeRequest) GetBips() (uint32, error) {
+	if msg.RecipientBasisPoints == "" {
+		return AssessCustomMsgFeeBips, nil
+	}
+	bips, err := strconv.ParseUint(msg.RecipientBasisPoints, 10, 32)
+	if err == nil {
+		if bips > 10_000 {
+			return 0, fmt.Errorf("recipient basis points can only be between 0 and 10,000 : %v", msg.RecipientBasisPoints)
```

Comment:
hardcoded max value in comments.

Reply:
Basis points should never be anything other than 0 - 10,000.  Having it hard coded allows me to format it as easily format it as `10,000`.

Reply:
👍